### ISSUE TITLE
[BUG][RESOLVED] SOGo Save Button Glitch Inside Mail Settings Section

### ISSUE THREAD

**Issue by Mr-Martinez** [mailcow/mailcow-dockerized#1697]
Hello,

I am experiencing a glitch with the save button in SOGo 4.0.1 . When I updated my mailcow server, I noticed that the save button would glitch out in the mail settings. The button behaves as it should when in general, calendar, and address book settings. However once you go to mail settings, the save button will change and stay in an active state while in the mail settings. When you leave the mail settings and head over to the another setting section, by clicking on the now active save button, it will deactivate and stay in a disabled state.

This issue is also apparent in new fresh installs of mailcow as well.

I would like to mention that I have had similar problems with this glitch about 1-2 weeks ago in SOGo when I tried to add a favicon into _Dockerfile_ found in _mailcow-dockerized/data/Dockerfiles/sogo_ and then building it with _docker-compose build sogo-mailcow_. 

This issue is not breaking but I would classify it as a bug.

Simulate Problem (est. time 8 minutes)
---------------------------------------
1. Digital Ocean VM Ubuntu 16.04 x64 
2. Install brand new instance of Mailcow
3. Create a single domain with a single user in the Mailcow UI
4. Login into SOGo with the user and head over to the settings page.
5. From general settings, head over to the mail preference settings.
6. Glitch should be apparent. Save button is not greyed out as it should and stays in an active state.

**Comment by hachre** [comment]
This sounds to me like a SOGo bug, you should try to also report it here:
https://sogo.nu/bugs/view_all_bug_page.php

**Comment by Mr-Martinez** [comment]
Hi Hachre,

I did find a solution to this bug.

For some reason after doing a fresh install, running the following codes found in the Mailcow Documentation for SOGo under the [Usage & Examples](https://mailcow.github.io/mailcow-dockerized-docs/u_e-sogo/) section cleared the bug away for me on the fresh install.

`docker-compose build sogo-mailcow`
`docker-compose up -d sogo-mailcow`

I can't explain how or why the issue is resolved though. 

I will close this and leave it if it helps others with this particular issue.

Running the above code also fixed the issue on my server that got updated as well.

It may as well be a SOGo bug. However based on the following, I don't think it is. Although I am very new to mailcow, github, and coding in general.

[Edit]
SOGo reports this as a [bug](https://sogo.nu/bugs/view.php?id=4528#c13003) for version 4.0.1 and has been fixed in version 4.0.2
